it/src/main/java/org/corfudb/universe/node/server/SupportServerParams.java

```diff
@@ -37,11 +35,12 @@
     @Default
     @NonNull
     @Getter
-    private final Level logLevel = Level.DEBUG;
+    private final Level logLevel = Level.INFO;
 
+    @Default
     @NonNull
     @Getter
-    private final NodeType nodeType;
+    private final NodeType nodeType = NodeType.METRICS_SERVER;
```

Comment:
SupportServer is meant for way more than just metrics. This being a default value makes no sense.